src/plots/mapbox/layout_attributes.js

```diff
@@ -0,0 +1,169 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var scatterMapboxAttrs = require('../../traces/scattermapbox/attributes');
+var defaultLine = require('../../components/color').defaultLine;
+var extendFlat = require('../../lib').extendFlat;
+
+var lineAttrs = scatterMapboxAttrs.line;
+
+
+module.exports = {
+    domain: {
+        x: {
+            valType: 'info_array',
+            role: 'info',
+            items: [
+                {valType: 'number', min: 0, max: 1},
+                {valType: 'number', min: 0, max: 1}
+            ],
+            dflt: [0, 1],
+            description: [
+                'Sets the horizontal domain of this subplot',
+                '(in plot fraction).'
+            ].join(' ')
+        },
+        y: {
+            valType: 'info_array',
+            role: 'info',
+            items: [
+                {valType: 'number', min: 0, max: 1},
+                {valType: 'number', min: 0, max: 1}
+            ],
+            dflt: [0, 1],
+            description: [
+                'Sets the vertical domain of this subplot',
+                '(in plot fraction).'
+            ].join(' ')
+        }
+    },
+
+    style: {
+        valType: 'string',
+        values: ['basic', 'streets', 'outdoors', 'light', 'dark', 'satellite', 'satellite-streets'],
+        dflt: 'basic',
+        role: 'style',
+        description: [
+            'Sets the Mapbox map style.',
+            'Either input the defaults Mapbox names or the URL to a custom style.'
+        ].join(' ')
+    },
+    center: {
+        lon: {
+            valType: 'number',
+            dflt: 0,
+            role: 'info',
+            description: 'Sets the longitude of the center of the map (in degrees East).'
+        },
+        lat: {
+            valType: 'number',
+            dflt: 0,
+            role: 'info',
+            description: 'Sets the latitude of the center of the map (in degrees North).'
+        }
+    },
+    zoom: {
+        valType: 'number',
+        dflt: 1,
+        role: 'info',
+        description: 'Sets the zoom level of the map.'
+    },
+    bearing: {
+        valType: 'number',
+        dflt: 0,
+        role: 'info',
+        description: 'Sets the bearing angle of the map (in degrees).'
+    },
+    pitch: {
+        valType: 'number',
+        dflt: 0,
+        role: 'info',
+        description: 'Sets the pitch angle of the map (in degrees).'
```

Comment:
Same here- it's probably going to be a little difficult to set bearing/pitch without some trial and error but I might add a little bit more here such as 0 is perpendicular to the surface of the map